manticore/utils/event.py

```diff
@@ -46,8 +46,9 @@ def _get_signal_bucket(self, name):
         #A bucket is a dictionary obj -> set(method1, method2...)
         return self._signals.setdefault(name,  dict())
 
-    def publish(self, name, *args, **kwargs):   
-        bucket = self._get_signal_bucket(name)
+    # The underscore _name is to avoid naming collisions with callback params
```

Comment:
Just if I wanted to have a `name=` kwarg in my callback handler. `self.publish('some_event', arg1, name='somename')` would fail because `name` would be passed twice.